src/cluster_legacy.h

```diff
@@ -377,6 +377,9 @@ struct clusterState {
      * stops claiming the slot. This prevents spreading incorrect information (that
      * source still owns the slot) using UPDATE messages. */
     unsigned char owner_not_claiming_slot[CLUSTER_SLOTS / 8];
+
+    /* Atomic slot migration */
+    list *asm_links;
```

Comment:
will a dest node be imported concurrently multiple slot ranges from different source nodes?